docs/docs/migration_notes.md

```diff
@@ -35,6 +35,29 @@ This PR also renamed encrypted events to private events:
 ### [Aztec.nr] Removal of `getSiblingPath` oracle
 Use `getMembershipWitness` oracle instead that returns both the sibling path and index.
 
+### [Aztec.nr] Introduction of `Packable` trait
```

Comment:
you might have to merge from master btw and move this to a new release section

Reply:
Would not have noticed this as it didn't cause conflicts. Thanks 🙏 

https://github.com/AztecProtocol/aztec-packages/pull/11136/commits/6dfa11042b57c92be6a9c8b0d35fe71a05f05039